### DISCUSSION TITLE
Dialog box for the problem moves on the X axis when selecting the lightbulb a second times

### DISCUSSION THREAD

**Discussion by lmcbout** [eclipse-theia/theia#7886] [General] [answered]
Tried with C++ and JAVA. When I have a lightbuld to select, when I select it one it opens a dialog box showing the Option to fix the issue. But when the dialog box is already open and I click on the lightbulb again, but a few pixels away, you see the dialog box moving on the "X" axis. Should we just keep the same position of the dialog when it is already open ?
Look at the dialog box under "Test" when clicking on the lightbuld a second time, the text is moving.

![DisplayBoxMoving](https://user-images.githubusercontent.com/419051/82838070-0805bc80-9e99-11ea-82fa-a00060afb390.gif)

**Comment by vince-fugnitto** [comment, marked as answer]
@lmcbout in my opinion this is a normal behavior, we trigger the rendering of the context-menu based on the x,y co-ordinates of the mouse (you're actually moving so the rendering anchor will be different).

**Comment by lmcbout** [comment]
Fine, but I think we should not re-anchor the dialog box position if it is already opened, i=t should stay at the same coordinates

**Reply by vince-fugnitto** [reply]
> Fine, but I think we should not re-anchor the dialog box position if it is already opened, i=t should stay at the same coordinates

@lmcbout this approach will cause more issues than it solves. There is nothing really to do here, vscode, browsers, applications all behave the same way, they use the new coordinates of to trigger the rendering of the context-menu.
